dashboard.php

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Dashboard</title>
+</head>
+<body>
+<header>
+        <nav>
+            <ul>
+                <li><a href="#">Home</a></li>
+                <li><a href="#">Stats</a></li>
+                <li><a href="#">Leaderboards</a></li>
+            </ul>
+        </nav>
+    </header>
+    <section>
+        <h2 class="section-title">Tests to complete</h2>
+            <div class="test-box">
+            <a href="testing-page.php" class="test-link">
+            <div class="test-contents">
+                <h3>Test Name</h3>
+                <p>Test Subject</P>
+            </div>
+            <div class="test-contents">
+                <h3>Test Name</h3>
+                <p>Test Subject</P>
+            </div>
+            <div class="test-contents">
+                <h3>Test Name</h3>
+                <p>Test Subject</P>
+            </div>
+            <div class="test-contents">
+                <h3>Test Name</h3>
+                <p>Test Subject</P>
+            </div>
+            </a>
+            </div>
+        </a>
+        
+    </section>
+    <section>
```

Comment:
I think the tests that the user has already completed should be hidden so they only see the tests they have left to do, but the completed tests could be revealed when a button is pushed.